catalog.yaml

```diff
@@ -293,6 +293,15 @@ topics:
         readme_url: https://raw.githubusercontent.com/cloudify-community/cloudify-catalog/master/terraform/aws/README.md
         main_blueprint: blueprint.yaml
         image_url: https://raw.githubusercontent.com/cloudify-community/cloudify-catalog/master/terraform/terraform-icon.jpeg
+      - id: terraform_azure_vpc
+        name: Azure VPC
```

Comment:
Name can't contain spaces, currently known bug in the UI``
```suggestion
        name: Azure_VPC
```

Reply:
corrected